Cargo.toml

```diff
@@ -3,3 +3,6 @@ members = [
   "src/core",
 ]
 default-members = ["src/core"]
+
+[profile.test]
+opt-level = 1
```

Comment:
I added this one in too, because the tests run MUCH faster (and compile time is about the same)